src/main.cpp

```diff
@@ -1,10 +1,77 @@
-#include <maya/MSimple.h>
-#include <maya/MIOStream.h>
+#include "plugin/ViewportRendererOverride.hpp"
 
-DeclareSimpleCommand(helloWorld, "Team Wisp", "2018/2019");
+#include "Defines.hpp"
 
-MStatus helloWorld::doIt(const MArgList&)
+#include <maya/MFnPlugin.h>
+
+#include <memory>
+
+static std::unique_ptr<wisp::ViewportRendererOverride> renderer_override_instance;
+
+void AddRemovePlugin(MStatus& t_status, bool t_add);
+void CheckStatus(const MStatus& t_status, const MString& t_message);
+
+// NOTE:	This function name deviates from the coding standard because Autodesk Maya looks for a function that is
+//			written exactly like this.
+MStatus initializePlugin(MObject t_object)
```

Comment:
Our style guide uses PascalCase not camelCase for functions

Reply:
I am aware of it, but Maya requires the function to be called "initializePlugin" and "uninitializePlugin", the way I understand it.